stores/sql/types.go

```diff
@@ -37,6 +37,7 @@ type (
 	FileContractID  types.FileContractID
 	Hash256         types.Hash256
 	MerkleProof     struct{ Hashes []types.Hash256 }
+	NullableInt64   int64
```

Comment:
Didn't we just get rid of this in a different PR and replaced it with just using `*int64`?